src/test/java/teammates/test/cases/action/AdminEmailComposePageActionTest.java

```diff
@@ -0,0 +1,85 @@
+package teammates.test.cases.action;
+
+import org.testng.annotations.Test;
+
+import teammates.common.datatransfer.attributes.AdminEmailAttributes;
+import teammates.common.util.Const;
+import teammates.logic.core.AdminEmailsLogic;
+import teammates.test.driver.AssertHelper;
+import teammates.ui.controller.AdminEmailComposePageAction;
+import teammates.ui.controller.ShowPageResult;
+import teammates.ui.pagedata.AdminEmailComposePageData;
+
+public class AdminEmailComposePageActionTest extends BaseActionTest {
+
+    private AdminEmailsLogic adminEmailsLogic = AdminEmailsLogic.inst();
+
+    @Override
+    protected void prepareTestData() {
+        dataBundle = loadDataBundle("/AdminEmailComposePageTest.json");
+        removeAndRestoreDataBundle(dataBundle);
+    }
+
+    @Override
+    protected String getActionUri() {
+        return Const.ActionURIs.ADMIN_EMAIL_COMPOSE_PAGE;
+    }
+
+    @Override
+    protected AdminEmailComposePageAction getAction(String... params) {
+        return (AdminEmailComposePageAction) gaeSimulation.getActionObject(getActionUri(), params);
+    }
+
+    @Override
+    @Test
+    public void testExecuteAndPostProcess() throws Exception {
+
+        final String adminUserId = "admin.user";
+        gaeSimulation.loginAsAdmin(adminUserId);
+
+        ______TS("case: compose new email");
```

Comment:
It seems `case:` doesn't add value. Perhaps you can use a format such as `scenario : expected behavior` instead?